CITATIONS.md

```diff
@@ -0,0 +1,54 @@
+# nf-core/chipseq: Citations
```

Comment:
It would be nice to pop the nf-core paper in here as well if you don't mind.

Reply:
Sure thing